stage1/init/container.go

```diff
@@ -43,6 +43,16 @@ type Container struct {
 	Networks           []string
 }
 
+var (
+	defaultEnv = [][2]string{
```

Comment:
No reason, looks like a C programmer wrote it, doesn't it?  I'll change it to a map.